main.qml

```diff
@@ -42,7 +42,7 @@ import "wizard"
 
 ApplicationWindow {
     id: appWindow
-
+    title: "Monero GUI"
```

Comment:
Maybe best to just use "Monero" for now until we have clear direction on other naming efforts?